### PR TITLE
InfluxDB: Added support for multiple queries and tags

### PR DESCRIPTION
**Pull request by groveriffic**
Multiple queries are separated by semicolons and could possibly have
different columns and tags.
https://docs.influxdata.com/influxdb/v0.11/guides/querying_data/#multiple-queries

Tags come in to the result set when you use a GROUP BY statement.

### CODE REVIEW

**Comment by arikfr** [comment]
Nice! Does it change the minimal version of Influxdb that we support?

**Comment by groveriffic** [comment]
I've been working against InfluxDB version 0.10.0.  I didn't see any info on minimum version support, could you point me in the right direction?

I'm also noticing that I've broken support for result sets *without* tags.  I should have that fixed shortly.

File: redash/query_runner/influx_db.py [hunk 1]
```diff
@@ -19,16 +19,32 @@ def _transform_result(results):
...
-            for c in result.raw['series'][0]['columns']:
-                result_columns.append({ "name": c })
+        for series in result.raw.get('series', []):
+            for column in series['columns']:
+                if column not in result_columns:
+                    result_columns.append(column)
+            tags = series.get('tags', {})
+            for key in tags.keys():
+                if key not in result_columns:
+                    result_columns.append(key)
```
Comment (arikfr) [inline redash/query_runner/influx_db.py]: Can't there by name collision between column names and tag names?

Comment (groveriffic) [inline redash/query_runner/influx_db.py]: Yes, that's possible and it causes problems when attempting to query based on the tag.

See also:
https://docs.influxdata.com/influxdb/v0.11/concepts/schema_and_data_layout/
bottom of the page.

**Comment by arikfr** [comment]
Merged. Thanks!

**Pull request merged**